### ISSUE TITLE
[Suggestion] Animate Java models

### ISSUE THREAD

**Issue by LoneDev6** [JannisX11/blockbench-plugins#96]
Hello! 
Would be really cool to implement this website as a plugin for Blockbench: https://vberlier.github.io/animated-models/
It allows us to animate models without having to switch custom_model_data like this: https://github.com/JannisX11/blockbench-plugins/blob/master/plugins/animator.js

This is an example video of this tool in action: https://www.youtube.com/watch?v=ZwdqY-7pYu4

Thanks for all your efforts

**Comment by LoneDev6** [comment]
Another link about this: https://twitter.com/MasterianoX/status/1187085961995390978

Would definitely be cool to have that feature implemented in Blockbench directly, it would save a lot of manual work.
Thanks for your time!

**Comment by JustiinGER** [comment]
That would be really awesome! I second this Idea!

**Comment by Jirsaacek** [comment]
Add this please! It would be really helpful! 😁

**Comment by hankwu0501** [comment]
awesome idea pls add it

**Comment by Command1264** [comment]
It's a good idea. Please add it.

**Comment by Agaloth** [comment]
Please add it, it's so good.

**Comment by donut4490** [comment]
ooh this would be neat

**Comment by spiker5032** [comment]
This would be a godsend!

**Comment by ArgonGaming** [comment]
Very nice suggestion

**Comment by 100petr** [comment]
It would be very COOL!

**Comment by SplendidBlossom** [comment]
Awesome Idea!

**Comment by jallen6769** [comment]
Please add this. I would be so excited to use it

**Comment by LoneDev6** [comment]
Hello! Is there any news about that feature idea? @JannisX11

**Comment by JannisX11** [comment]
> Hello! Is there any news about that feature idea? @JannisX11

No, I don't think anyone has picked up this idea. There is however a different plugin for creating animations for Minecraft: Java Edition that uses a different approach. Is is not fully released yet, but you can learn more about it here: https://twitter.com/SnaveSutit/status/1345139043105959937

**Comment by LoneDev6** [comment]
> 
> 
> > Hello! Is there any news about that feature idea? @JannisX11
> 
> No, I don't think anyone has picked up this idea. There is however a different plugin for creating animations for Minecraft: Java Edition that uses a different approach. Is is not fully released yet, but you can learn more about it here: https://twitter.com/SnaveSutit/status/1345139043105959937

Thanks for the info, sadly I already know this method and it's not really good as it seems because of the high server side CPU usage.

The method I showed is entirely client side which allows tons of mobs to be spawned without seeing servers explode.
Do you think that it would be hard to implement? I have no knowledge on how blockbench plugins are structured.

Here is the [source code](https://github.com/vberlier/animated-models) of an already existing tool, would be very handy to have it integrated into blockbench.

Thanks for your hard work

**Comment by IanSSenne** [comment]
you can probably learn the basic structure of a plugin from the plugins in this repo or feel free to drop by the discord mentioned above and I would be happy to help you get started 😺, we are working hard to reduce the impact added to the server from armor stands to make this viable. sadly you are right about not being able to spawn a huge number of them without adding a bit of server strain.

**Comment by DoubleF3lix** [comment]
Well... it's published.
